src/containers/Transactions/simpleTab.scss

```diff
@@ -94,6 +94,14 @@ $subdued-color: $black-40;
       background: rgba($black-80, 0.7);
       gap: 15px;
 
+      &:first-child {
```

Comment:
I would think gap would be accomplishing this.

Reply:
Apparently not /shrug